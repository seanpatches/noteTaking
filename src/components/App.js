import React from 'react';
import CreateNote from '../container/CreateNote.js';
import NoteDetail from '../container/NoteDetail';
import AllNotes from '../container/AllNotes';
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';

export default function App() {
  return (
    <Router>
      <Switch>
        <Route exact path='/' component={AllNotes}/>;
        <Route path='/:title' component={NoteDetail} />;
      </Switch>
    </Router>
  );
}
