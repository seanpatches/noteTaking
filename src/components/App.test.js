import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
jest.mock('../services/auth.js');

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
