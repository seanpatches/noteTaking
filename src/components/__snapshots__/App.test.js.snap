// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(withSession)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    ;
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Callback)",
          "type": [Function],
        }
      }
      exact={true}
      path="/callback"
    />
    ;
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(withSession)",
          "type": [Function],
        }
      }
      path="/:id"
    />
    ;
  </Switch>
</BrowserRouter>
`;
